### PR TITLE
rgw: update outdated debug func name

### PR DESCRIPTION
**Pull request by mikulely**

### CODE REVIEW

**Comment by mikulely** [comment]
@joscollin @cbodley mind to merge?

File: src/rgw/rgw_rados.cc [hunk 1]
```diff
@@ -10645,13 +10645,13 @@ int RGWRados::Object::Read::iterate(int64_t ofs, int64_t end, RGWGetDataCB *cb)
   while (!done) {
     r = data->wait_next_io(&done);
     if (r < 0) {
-      dout(10) << "get_obj_iterate() r=" << r << ", canceling all io" << dendl;
+      dout(10) << __func__ << " r=" << r << ", cancelling all io" << dendl;
```
Comment (cbodley) [inline src/rgw/rgw_rados.cc]: nit: `canceling` is also a correct spelling, so i'd prefer not to change it

Comment (joscollin) [reply src/rgw/rgw_rados.cc]: @cbodley `canceling` is American and `cancelling` is British. So for people on the Eastern side of the world `cancelling` looks correct :-D
https://english.stackexchange.com/questions/42745/canceling-or-cancelling

I think, let's follow American, as cbodley suggests.

**Comment by mikulely** [comment]
Jenkins, retest this please.

**Comment by joscollin** [comment]
[wip-jcollin-testing-10-17-2017](https://trello.com/c/2Gvg9mRo/180-wip-jcollin-testing-10-17-2017)

**Pull request merged**